edx_exams/apps/api/v1/views.py

```diff
@@ -358,28 +358,25 @@ def get_response(cls, exam, user):
         elif exam.due_date is not None and timezone.now() >= exam.due_date:
             data, grant_access, response_status = {}, True, status.HTTP_200_OK
 
-        response = Response(status=response_status,
-                            data=data)
-
         if grant_access:
             log.info("Creating exam access token")
             access_token = sign_token_for(user.lms_user_id, expiration_window, claims)
-            response.set_cookie(
-                settings.ACCESS_TOKEN_COOKIE_NAME, access_token,
-                domain=settings.ACCESS_TOKEN_COOKIE_DOMAIN
-            )
+            data = {ACCESS_TOKEN_COOKIE_NAME: access_token, ACCESS_TOKEN_COOKIE_NAME + "_expiration": expiration_window}
+
+        response = Response(status=response_status,
+                            data=data)
 
         return response
 
-    def get(self, request, exam_id):
+    def get(self, request, course_id, content_id):
         """
         Get exam access token as JWT added as cookie to response.
 
         Exam access token corresponds to given exam and user in request.
         """
-        try:
-            exam = Exam.objects.get(id=exam_id)
-        except ObjectDoesNotExist:
+        exam = get_exam_by_content_id(course_id, content_id)
```

Comment:
[Question] I noticed while creating unit tests that get_exam_by_content_id can return multiple exams if the content_ids are duplicates. Is this being checked/prevented somewhere else? In either case, would it be better to have a check in this method to make sure that there aren't duplicates? And to handle the error if so?

Reply:
Ah okay. And to clarify, is it possible to have duplicates by content_id? Or is this check not needed?

Reply:
it shouldn't be possible to create duplicates so maybe we want a `unique_together` constraint on (course_id, content_id, is_active).

Reply:
That sounds good!

Reply:
Since we are keeping the old endpoint now, I will create a separate bug ticket to fix this bug